tests/parsers_test.py

```diff
@@ -9,7 +9,7 @@
 
 SAMPLES_PATH = os.path.join(os.path.dirname(__file__), "samples")
 SAMPLES = {}
-for extension in [".spe", ".spc", ".cnf"]:
+for extension in [".spe", ".spc", ".cnf", ".hdf5"]:
```

Comment:
And `.h5` as that is more common 😄

Reply:
This was written for our particular cases, although I suppose I'm the one who created the file at issue (Mendocino*.hdf5).